components/google-cloud/container/aiplatform/Dockerfile

```diff
@@ -21,6 +21,6 @@ WORKDIR /root
 RUN pip3 install --upgrade pip
 
 # TODO() change from using PR commit id to release 
-RUN pip3 install "git+https://github.com/kubeflow/pipelines.git#egg=google-cloud-pipeline-components&subdirectory=components/google-cloud"
+RUN pip3 install "git+https://github.com/kubeflow/pipelines.git@refs/pull/5721/merge#egg=google-cloud-pipeline-components&subdirectory=components/google-cloud"
```

Comment:
I don't believe this is necessary, once the PR is merged to head the master will be at this PR or ahead of it.

Reply:
/unhold

Reply:
removed the hold since the change is harmless